modules/apps/portal-workflow/portal-workflow-kaleo-runtime-impl/src/main/java/com/liferay/portal/workflow/kaleo/runtime/internal/notification/recipient/RoleNotificationRecipientBuilder.java

```diff
@@ -121,6 +121,29 @@ protected void deactivate() {
 		_serviceTrackerList.close();
 	}
 
+	protected boolean isSelfAssignedUser(
```

Comment:
Hi @vngoliveira , I understand that this method was turned `protected` only to fit in the test. If that is the case we can use the `Method` class like what is done in `ObjectEntryEntityModelTest.java` to keep this method `private`.